Core.Tests/StateManagerTests.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Xunit;
+
+namespace Core.Tests
+{
+    public class StateManagerTests
+    {
+        [Fact]
+        public void ReturnsTrue()
+        {
+            Assert.True(true);
+        }
+
+        [Fact]
+        public void Increment()
+        {
+            var res = State.Increment(5);
+            Assert.Equal(6, res);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '6' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931948)